docs/vector.rst

```diff
@@ -227,17 +272,31 @@ Metadata
    :parameter a: (ti.Vector or ti.Vector.field)
    :return: (scalar) return the dimensionality of vector ``a``
 
-    E.g.,
-    ::
+   E.g.,
+   ::
 
         # Taichi-scope
         a = ti.Vector([1, 2, 3])
         a.n  # 3
 
-    ::
+   ::
 
         # Python-scope
-        a = ti.Vector.field(3, dtype=ti.f32, shape=())
+        a = ti.Vector.field(3, dtype=ti.f32, shape=(4, 5))
         a.n  # 3
 
+.. note::
+
+   When used as a global vector field, it will additionally contain all the
```

Comment:
```suggestion
   When this attribute is used in a global vector field, it will additionally contain all the
```
Maybe specifying the subject can make it more clear, HDYT?

Reply:
In fact the subject is `this class`, i.e. `ti.Vector`.